### PR TITLE
Describe compiled css

### LINKED ISSUE #24262: Docs: Describe compiled CSS files in download bundle
I downloaded the pre-compiled bootstrap-4.0.0-beta-dist.zip and it includes three flavors of CSS files:

- bootstrap-grid.css
- bootstrap-reboot.css
- bootstrap.css

It didn't seem evident from a few searches whether I needed all 3 files, whether they referenced each other with `@import`, or what. The only reference I can find in the documentation is this on the Download page (emphasis mine):

> Download ready-to-use compiled code for Bootstrap v4.0.0-beta to easily drop into your project, which includes:
> - Compiled and minified CSS bundles (**default, grid-only, and Reboot-only**)
> - Compiled and minified JavaScript plugins

These references cannot be found by search, since they don't match the file name.

It would be nice if the documentation went into more detail about these.

### PR DESCRIPTION
**Pull request by ghost**
Fixes #24262

### CODE REVIEW

**Comment by lucascono** [comment]
@XhmikosR: My God, I'm going crazy. So I will not be able to make any other contributions until I decide what I did in the master?

**Comment by lucascono** [comment]
Reject this @XhmikosR. For today I lost my patience. Sorry.

**Comment by XhmikosR** [comment]
It's pretty simple, no need to lose your cool :P

```
git fetch upstream
git checkout describe-compiled-css
git rebase -i upstream/v4-dev
```

and drop the unrelated patches.

File: docs/4.0/getting-started/contents.md [hunk 1]
```diff
@@ -18,14 +18,70 @@ bootstrap/
...
+│   ├── bootstrap-reboot.min.css
+│   └── bootstrap-reboot.min.css.map
 └── js/
     ├── bootstrap.js
     └── bootstrap.min.js
 {% endhighlight %}
 
 This is the most basic form of Bootstrap: precompiled files for quick drop-in usage in nearly any web project. We provide compiled CSS and JS (`bootstrap.*`), as well as compiled and minified CSS and JS (`bootstrap.min.*`). CSS [source maps](https://developers.google.com/web/tools/chrome-devtools/javascript/source-maps) (`bootstrap.*.map`) are available for use with certain browsers' developer tools.
 
+### Comparative of CSS files
```
Comment (XhmikosR) [inline docs/4.0/getting-started/contents.md]: Comparison. Also change the link that references the anchor.

**Review by XhmikosR — CHANGES_REQUESTED** [review]

**Pull request closed without merging**